### DISCUSSION TITLE
Documentation update

### DISCUSSION THREAD

**Discussion by mhwasil** [ltsp/ltsp#246] [Q&A]
The HOSTNAME only works when it´s configured under [client], while in the default ltsp.conf it´s under [mac:address]

**Comment by alkisg** [comment]
HOSTNAME should work fine under [mac:address]. Of course, you need a real MAC address (lowercase) there. What's your ltsp.conf like?

Example:

```shell
[b8:27:eb:89:17:56]
HOSTNAME=rpi3
```

**Comment by alkisg** [comment]
Moved to community as I think this wasn't an issue.

**Comment by mhwasil** [comment]
Sorry for slow response, it was examination period and I was little bit busy with students, but this is done now.
So, of course I put correct mac address and put the HOSTNAME under the mac address, but it does not work, it gives me the default name ''ltsp101'' for example.
But when I put it under [client], then it works.

Here is my config:

```
[server]
# Enable NAT on dual NIC servers
# NAT=1
# Provide a full menu name for x86_32.img when `ltsp ipxe` runs
# IPXE_X86_32_IMG="Debian Buster"
DEBUG_SHELL=1
DEFAULT_IMAGE="x86_64"
#DEFAULT_IMAGE="debian"
#DEFAULT_IMAGE="debian-live"

# The special [common] section is evaluated by both the server and ltsp clients
[common]
# Specify an alternative TFTP_DIR
# TFTP_DIR=/var/lib/tftpboot
LIGHTDM_CONF="greeter-hide-users=true"
MENU_TIMEOUT="-1"

# In the special [clients] section, parameters for all clients can be defined.
# Most ltsp.conf parameters should be placed here.
[clients]
# Specify an /etc/fstab line for NFS home; note this is insecure
# FSTAB_HOME="server:/home /home nfs defaults,nolock 0 0"
IMAGE_TO_RAM=0    # copy image to ram during boot (this makes client less dependent on server
HOSTNAME="student%{IP}"

# MAC address, IP address, or hostname sections can be used to apply settings
# to specific clients.
[9c:7b:ef:15:a5:22]
# HOSTNAME=pc01
# Include parameters from another section, defined below
# INCLUDE=crt_monitor

.
 ```

**Comment by alkisg** [comment]
Hi, try this instead:

```shell
[clients]
# Comment this out:
#HOSTNAME="student%{IP}"

[9c:7b:ef:15:a5:22]
# And uncomment this one:
HOSTNAME=pc01
```

**Comment by mhwasil** [comment]
using HOSTNAME=pc01 does not work too,
The user hostname becomes the default "ltsp101" again

**Comment by alkisg** [comment]
@mhwasil, try to concentrate on discovering what you're doing wrong, as HOSTNAME under [mac] surely works.

Did you run `ltsp initrd` after modifying ltsp.conf?
Is the [mac:address] correct?
If you put anything else under [mac], like `POST_INIT_SHELL=bash`, does it work, does the client give you a bash shell with that?

**Comment by mhwasil** [comment]
I think I am misunderstood here. What I want is that the clients should have ''students$ip'',
which is working for all clients if I put it under [client].

The other setting under [mac] is for specific client, and that should be client mac address.

It's all fine now.

So I will just close the issue

Thank you for the support
